pkg/aws/ocpgwdeployer.go

```diff
@@ -142,6 +151,23 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 	return nil
 }
 
+func (d *ocpGatewayDeployer) setSubnetSuffixes(publicSubnets []types.Subnet) {
+	pattern := fmt.Sprintf(`%s.*-subnet-public-%s.*`, regexp.QuoteMeta(d.aws.infraID), regexp.QuoteMeta(d.aws.region))
+	re := regexp.MustCompile(pattern)
+
+	for i := range publicSubnets {
+		tags := publicSubnets[i].Tags
+		for i := range tags {
+			if strings.Contains(*tags[i].Key, "Name") {
+				if re.MatchString(*tags[i].Value) {
```

Comment:
maybe we can change it to : 

`if strings.Contains(*tags[i].Key, "Name") && re.MatchString(*tags[i].Value) {

}` ?

Reply:
Done.